### PR TITLE
[0.26] Describe new automatic client name and address attributes, per-client subscriptions

### LINKED ISSUE #417: Document new "individual" client subscriptions (0.26)
See: https://github.com/sensu/sensu/issues/1327

### LINKED ISSUE #420: Note that client "name" and "address" may be auto discovered (0.26)
See: https://github.com/sensu/sensu-settings/pull/37

### PR DESCRIPTION
**Pull request by cwjohnston**
Closes #420 
Closes #417

### CODE REVIEW

File: docs/0.26/reference/clients.md [hunk 1]
```diff
@@ -387,7 +395,9 @@ The client definition uses the `{ "client": {} }` [configuration scope][24].
   : An address to help identify and reach the client. This is only
     informational, usually an IP address or hostname.
 : required
-  : true
+  : false
+: default
+  : Non-loopback IPv4 address as determined by `Socket.ip_address_list`
```
Comment (calebhailey) [inline docs/0.26/reference/clients.md]: Should this say "by Ruby `Socket.ip_address_list`" (+Ruby), like above?

**Comment by calebhailey** [comment]
Other than one minor suggestion, this LGTM!

**Comment by cwjohnston** [comment]
@calebhailey minor suggestion acted upon 👍

**Comment by calebhailey** [comment]
LGTM! :+1:

**Pull request merged**